draft-ietf-sedate-datetime-extended.md

```diff
@@ -255,15 +253,15 @@ By including a hyphen/minus sign "`-`", the namespace can be separated
 from the rest of the key; if no hyphen/minus sign is included, the
 whole key is the namespace.
 
-For example, if "`u-`" is a namespace for the Unicode consortium, a
-calendar as defined by that consortium could be included as
+For example, if "`u-`" is a namespace for the Unicode Consortium, a
+calendar as defined by that organization could be included as
 `u-ca=<value>`.
 
 An IANA registry for namespaces can be used to allocate namespaces for
 specific applications, as defined in {{iana-cons}}.  Two namespaces are
-allocated by the present document:
+allocated by this document:
```

Comment:
I generally don't know what "this document" refers to.
Can we be more explicit?

Reply:
"This document" is fine unless it isn't.  This passage refers to a registry, which (loosely speaking) is a document.  So "this document" is ambiguous (and actually preferably refers to any referent that may be in scope, which is the wrong reference here) and needs to be disambiguated.  Changing the other 13 instances to also be unambiguous may be needed if a candidate reference is in scope, but it is this one that is clearly confusing to this reader.

Reply:
I've updated this to clearly reference the registry, although I think more dramatic refactoring is in order to initialize the registry with more formal entries that specify all fields of the included template as in e.g. [RFC 7763](https://www.rfc-editor.org/rfc/rfc7763.html#section-6) and [RFC 5545](https://datatracker.ietf.org/doc/html/rfc5545#section-8).